### DISCUSSION TITLE
LCF feature

### DISCUSSION THREAD

**Discussion by mrvejar** [xraypy/xraylarch#420] [Ideas]
Hello, I am curious whether an option could be added to the LCF Array to fit. Specifically, if there could be an option to fit the "raw" array.

**Comment by newville** [comment]
@mrvejar Using "raw" data probably could be added, but I don't think that it is a great idea.  An assumption of LCF is that all the datasets are properly normalized to start with.   Using un-normalized data would give very questionable results.   But, maybe I am not understanding what you are trying to do.  

Why do you want to do LCF with unnormalized data?  What are you hoping to learn from that?

**Reply by mrvejar** [reply]
Thank you for your response @newville . I have normalized data (normalized in PyMca) that does not have an edge step (actinide M-edge data). I have tried manually "normalizing" this M-edge data for LCF. However, sometimes the absorbance in the post-edge region is higher than the pre-edge absorbance, which makes it a very manual process to apply evenly.

**Comment by newville** [comment]
@mrvejar   Just to be clear, can you verify that you are talking about doing Linear Combination Fitting for XANES data with XAS Viewer?   What you are saying is very confusing to me.

I have no idea what "normalized in PyMca" means. I can definitely guess that it is very different from what we mean here for XANES data.   
The absorbance in the post-edge region **should** be large that in the pre-edge region.   If it is XAS data, it has an edge step.   For actinide M edges, that might be tiny compared to the height of the white line, but it is not zero.   

With unnormalized XANES data, Linear Combination fits will not give meaningful results.

**Reply by mrvejar** [reply]
@newville Yeah I am talking about doing Linear Combination Fitting for XANES data with XAS viewer, thank you for guiding me to be more specific. Thank you for the clarification that actinide M edges still have an edge step, I think this was the source of my confusion. 

I definitely see your point and join you in agreement that this data should still be normalized prior to Linear Combination fitting. Thank you so much.
